client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -81,10 +83,67 @@ describe('FeedbackGeneratorService', function() {
             c: 'j'
           })
         ).toEqual('{"a": 3, "b": 5, "c": "j"}');
+        // Define and call blankFunction to pass lint test add 100% coverage
```

Comment:
This might be worth pulling out into its own separate test, like it(returns [UNKNOWN OBJECT] if the provided variable cannot be properly converted) or something